frontend/packages/metal3-plugin/src/components/dashboard/health-card.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+
+import { DashboardItemProps } from '@console/internal/components/dashboards-page/with-dashboard-resources';
+import { DashboardCard } from '@console/internal/components/dashboard/dashboard-card';
+import { DashboardCardBody } from '@console/internal/components/dashboard/dashboard-card/card-body';
+import { DashboardCardHeader } from '@console/internal/components/dashboard/dashboard-card/card-header';
+import { DashboardCardTitle } from '@console/internal/components/dashboard/dashboard-card/card-title';
+import { HealthBody } from '@console/internal/components/dashboard/health-card/health-body';
+import { HealthItem } from '@console/internal/components/dashboard/health-card/health-item';
+import { HealthState } from '@console/internal/components/dashboard/health-card/states';
+
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import {
+  HOST_STATUS_OK,
+  HOST_HEALTH_OK,
+  HOST_STATUS_ERROR,
+  HOST_HEALTH_ERROR,
+  HOST_HEALTH_LOADING,
+} from '../../constants';
+import { getHostOperationalStatus } from '../../selectors';
+
+const getHostHealthState = (obj): HostHealthState => {
+  const status = getHostOperationalStatus(obj);
+
+  switch (status) {
```

Comment:
I think this will have to apply on all success states and error states:
```
switch (true) {
  case HOST_SUCCESS_STATES.includes(status):
    return {...
  case HOST_ERROR_STATES.includes(status):
   return {...
 default:
    ...
```